doc/api/stream.md

```diff
@@ -2517,6 +2517,10 @@ run().catch(console.error);
 after the `callback` has been invoked. In the case of reuse of streams after
 failure, this can cause event listener leaks and swallowed errors.
 
+`steam.pipeline()` closes all the streams when an error is raised. It means that
+`IncomingRequest` should be avoided once it would destroy the socket without
+sending any response.
```

Comment:
I don't think we should state `IncomingRequesti` should be avoided, rather I think we should use it as an example of what to keep in mind.

Reply:
Fair enough. I'll add an example.

Reply:
It's better now?